### PR TITLE
Collapse consecutive rename messages

### PR DESCRIPTION
**Pull request by AnnikaCodes**

### CODE REVIEW

File: src/battle-log.ts [hunk 1]
```diff
@@ -133,11 +139,23 @@ class BattleLog {
...
+			if (toID(args[2]) === toID(user.name)) return;
+			if (!this.lastRename) {
+				this.lastRename = {
+					from: args[2],
+					to: '',
+					element: document.createElement('div'),
+				};
+				this.lastRename.element.className = 'chat';
+			} else if (toID(this.lastRename.to) !== toID(user.name)) {
+				this.lastRename.from = args[2];
```
Comment (Zarel) [inline src/battle-log.ts:151]: ? Was this a mistake?

Comment (Zarel) [inline src/battle-log.ts:151]:
```suggestion
			if (!this.lastRename || toID(this.lastRename.to) !== toID(user.name)) {
				this.lastRename = {
					from: args[2],
					to: '',
					element: document.createElement('div'),
				};
				this.lastRename.element.className = 'chat';
```

This seems like what you'd want? Unless you intentionally wanted to hide earlier renames if multiple people renamed in a row.

Comment (Zarel) [reply src/battle-log.ts:151]: I suppose it wouldn't be horrible if you did, but I don't know if that's necessary.

Comment (AnnikaCodes) [reply src/battle-log.ts:151]: Mm, I suppose multiple users renaming would be pretty difficult to abuse with a bot.

**Pull request merged**